Documentation/coding-guidelines/breaking-changes.md

```diff
@@ -16,47 +22,86 @@ To help triage breaking changes, we classify them in to four buckets:
 3. Unlikely Grey Area
 4. Clearly Non-Public
 
-### Bucket 1: Public Contract
+## Bucket 1: Public Contract
 *Clear violation of public contract.*
 
 Examples:
-* throwing a new/different exception type in an existing common scenario
```

Comment:
💭 This would still apply if the exception type and scenario is explicitly documented